### DISCUSSION TITLE
Welcome to club-iEES Discussions!

### DISCUSSION THREAD

**Discussion by JeromeMathieuEcology** [JeromeMathieuEcology/club-iEES#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rosalieb** [comment]
Test! Anyone can post.

**Comment by rosalieb** [comment]
Hello Jerome,
I don't know if this is the right place to post this, but just to let you know that on Wednesday June 1st, there will be a labchat at 2PM (right after the Club Stats). A visiting researcher for U Davis will be speaking, and she couldn't make it on Thursday during the regular time slot.
So just FYI that we will be needing the room for that at 2PM!
Thank you,
Rosalie

**Reply by JeromeMathieuEcology** [reply]
Ok no prob! Thanks for the notice.
Jerome
